pyproject.toml

```diff
@@ -62,3 +62,6 @@ include = "(^/src/.*.py|^/tests/.*.py)"
 profile = "black"
 line_length = 119
 skip = ["__init__.py"]
+
+[tool.pytest.ini_options]
+norecursedirs = []  # overwrite the default because per default the test_build folder is skipped
```

Comment:
```suggestion
norecursedirs = []  # overwrite the default to not skip tests/build folder which is needed in spdx3 prototype
```

Reply:
changed. I omitted the "prototype" as we will probably forget to erase that later when it is not a mere prototype anymore